ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -39,6 +39,13 @@ azure_connection::azure_connection(const std::string &bucket_name, const std::st
         std::getenv("AZURE_STORAGE_CONNECTION_STRING"), bucket_name)),
       _bucket_name(bucket_name), _object_prefix(obj_prefix)
 {
+    // Confirm that we can access the bucket, else fail.
+    bool exists;
+    int ret = bucket_exists(exists);
+    if (!exists)
+        std::cerr << _bucket_name + " : No such bucket." << std::endl;
+    if (ret != 0)
+        std::cerr << _bucket_name + " : Unable to access bucket." << std::endl;
```

Comment:
ditto

Reply:
done :)

Reply:
same as above

Reply:
reverted - cant return value in constructor

Reply:
closing this due to impl of exceptions